apps/aecore/test/aecore_chain_state_test.exs

```diff
@@ -37,11 +38,9 @@ defmodule AecoreChainstateTest do
   test "chain state", wallet do
     init_accounts_state = Chain.chain_state().accounts
 
-    {:ok, signed_tx1} =
-      Account.spend(wallet.b_pub_key, wallet.b_priv_key, wallet.a_pub_key, 1, 1, 2, <<"payload">>)
-
-    {:ok, signed_tx2} =
-      Account.spend(wallet.c_pub_key, wallet.c_priv_key, wallet.a_pub_key, 2, 1, 2, <<"payload">>)
+    Account.spend(wallet.b_pub_key, wallet.b_priv_key, wallet.a_pub_key, 1, 1, 2, <<"payload">>)
+    Account.spend(wallet.c_pub_key, wallet.c_priv_key, wallet.a_pub_key, 2, 1, 2, <<"payload">>)
+    txs = Pool.get_pool() |> Map.values()
```

Comment:
While testing the miner is running in the background, so they could be gone already, the old way looked better

Reply:
I'm pretty sure the miner isn't running in background on tests, but I agree.

Reply:
okay, I am not to sure about this, but it should be adjusted anyways